packages/stacks/src/mocks/mocks.ts

```diff
@@ -0,0 +1,8 @@
+// Stacks test addresses
+export const TEST_ACCOUNT_1_STX_ADDRESS = 'SPS8CKF63P16J28AYF7PXW9E5AACH0NZNTEFWSFE';
+export const TEST_ACCOUNT_2_STX_ADDRESS = 'SPXH3HNBPM5YP15VH16ZXZ9AX6CK289K3MCXRKCB';
+export const TEST_TESTNET_ACCOUNT_2_STX_ADDRESS = 'STXH3HNBPM5YP15VH16ZXZ9AX6CK289K3NVR9T1P';
+// TODO ask about these prefixs
```

Comment:
@fbwoolf do you know about the different STX address types? 

We have a check - `isValidAddressChain` and I wanted to add tests to make sure we correctly validate `SM` and `SN` addresses. I tried to hack them by changing the prefix of other addresses but then they are invalid addresses.

Reply:
I believe `SM` and `SN` are mainnet/testnet for multisig?

Reply:
Reference here: https://github.com/stacksgov/sips/blob/2c3b36c172c0b71369bc26cffa080e8bdd2b3b84/sips/sip-005/sip-005-blocks-and-transactions.md#transaction-authorization

<img width="1059" alt="Screenshot 2025-02-20 at 9 53 20 AM" src="https://github.com/user-attachments/assets/31544eed-8895-41c9-b820-e19853e45ddf" />

Reply:
We do need to support multisig, and I believe it could be for other things too, this is an example of the one being used to deploy sbtc contracts: https://explorer.hiro.so/address/SM3VDXK3WZZSA84XXFKAFAF15NNZX32CTSG82JFQ4?chain=mainnet

Reply:
I haven't tested, but I wonder if Asigna produces these addresses? 🤔 https://www.asigna.io/

Reply:
Thanks a lot Fara! That will help me cover it and learn more about it

Reply:
I'll add these extended tests in my next PR. Just to be clear, we do still support multisig as ever, I just want us to have better tests